BUILDING.md

```diff
@@ -48,3 +48,15 @@ To build without executing tests:
 To find the Java binary in your environment, run the appropriate command for your operating system:
 * Linux/macOS: `which java`
 * Windows: `for %i in (java.exe) do @echo. %~$PATH:i`
+
+## Neon.3 Release
+Note that the following has been tested on Eclipse Neon.3 Release (4.6.3). Neon.3 already includes the required m2e and m2e-wtp plugins.
+
+The steps above still work to build without executing tests.
+
+There will be a few build errors in Eclipse Package Explorer. To fix the errors and debug JanusGraph code in Eclipse (breakpoints, step into/over, etc):
+
+1. Add `<your_janusgraph_home>/janusgraph-test/target/generated-sources/groovy-stubs/main` to the Java build path for `janusgraph-test` project
+2. Change Maven’s errors/warnings "Plugin execution not covered by lifecycle configuration" setting to "Ignore"
+3. Comment out the dependency for `janusgraph-hadoop` in the `pom.xml` for `janusgraph-all` project
```

Comment:
I haven't tried it out for myself yet with Eclipse, but commenting out a dependency seems like a hack -- a hack that could likely get checked in by mistake. Any insight on why that particular dependency is messing up Eclipse?